### PR TITLE
[BH-1246] PowerNap Frontlight Functionality

### PR DESCRIPTION
**Pull request by tomaszrybarski-digica**
Frontlight is enabled on Powernap finish

### CODE REVIEW

File: products/BellHybrid/apps/application-bell-powernap/presenter/PowerNapProgressPresenter.cpp [hunk 1]
```diff
@@ -56,6 +57,14 @@ namespace app::powernap
     }
     void PowerNapProgressPresenter::onNapFinished()
     {
+        const std::string frontlightEnabled =
+            settings->getValue(bell::settings::Alarm::lightActive, settings::SettingsScope::Global);
+
+        if (frontlightEnabled == std::string("1")) {
```
Comment (swichu91) [inline products/BellHybrid/apps/application-bell-powernap/presenter/PowerNapProgressPresenter.cpp:63]: There is an `AlarmLightOnOffModel` you can use to obtain that value. It probably should be moved to the `common` too.

**Review by swichu91 — APPROVED** [review]
LGTM, but please check the comment.

**Review by mjan-dev — APPROVED** [review]
Please consider @swichu91 comment before merging.

File: products/BellHybrid/apps/application-bell-powernap/presenter/PowerNapProgressPresenter.cpp (hunk 1 above)
Comment (tomaszrybarski-digica) [reply products/BellHybrid/apps/application-bell-powernap/presenter/PowerNapProgressPresenter.cpp]: Added AlarmLightOnOffModel to PowerNapProgressPresenter

File: products/BellHybrid/apps/application-bell-powernap/presenter/PowerNapProgressPresenter.cpp [hunk 2]
```diff
@@ -71,6 +75,11 @@ namespace app::powernap
 
     void PowerNapProgressPresenter::onNapFinished()
     {
+        if (alarmLightOnOffModel->getValue() == true) {
```
Comment (swichu91) [inline products/BellHybrid/apps/application-bell-powernap/presenter/PowerNapProgressPresenter.cpp:78]:
```suggestion
        if (alarmLightOnOffModel->getValue()) {
```

Comment (tomaszrybarski-digica) [reply products/BellHybrid/apps/application-bell-powernap/presenter/PowerNapProgressPresenter.cpp]: Done

**Pull request merged**